starter-code/public/scripts/articleView.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 var app = app || {};
 
+(function(module){
```

Comment:
you forgot to indent the contents of articleView after you wrapped them in the IIFE so you just need to indent everything between what you wrapped by one.